### ISSUE TITLE
/oper command is missing

### ISSUE THREAD

**Issue by mrhmouse** [emacs-circe/circe#238]
The `/oper` command from [RFC 1459](https://tools.ietf.org/html/rfc1459#section-4.1.5) is missing. This is useful for e.g. authenticating with [bitlbee](https://github.com/bitlbee/bitlbee/).

**Comment by mrhmouse** [comment]
It would be nice to mask the password portion of this command on-screen, but that probably belongs in a separate issue.

**Comment by jorgenschaefer** [comment]
Indeed, Circe is missing a few commands – thanks for the suggestion! Shouldn't be a problem at all to not display the password, though. This requires defining a command like `circe-command-OPER`.

**Comment by wasamasa** [comment]
The command lives in circe-chanop.el, so a `(require 'circe-chanop)` is necessary to use it. Successfully tested with Bitlbee.
